auslib/test/web/test_client.py

```diff
@@ -907,6 +908,54 @@ def testNonSubstitutedUrlVariablesReturn404(self):
                 self.assertEqual(ret.status_code, 404)
                 self.assertFalse(mock_cr_view.called)
 
+    def testUpdateQueryVersion(self):
+        version_key = 'queryVersion'
+        url_params = {}
+        update_query_version('/update/1/a/b/c.xml', url_params)
```

Comment:
This looks like an excellent use for hypothesis! Rather than writing all of these out by hand, you can apply a decorator like:
```
@given(integers(min_value=1, max_value=X))
def testUpdateQueryVersion(self, qv):
```

...which will set ```qv``` to numerous different values for you.